docs/projects/amt/adrs/0008-systemcard-storage.md

```diff
@@ -0,0 +1,53 @@
+# AMT-0008 System card Storage
+
+## Context
+
+By default, Kubernetes pods use ephemeral storage, which is tied to the pod's lifecycle.
+When the pod terminates or restarts, all data is lost. The `/tmp/` directory, being part of
+the system's temporary file storage, is cleared during reboots or pod restarts, resulting in
+the deletion of system_cards. Therefore, we need a different kind of storage to preserve the data.
+
+## Assumptions
+
+* The system card data is small to moderate in size (up to 2GB), making it manageable for SQLite.
```

Comment:
I think I saw somewhere online that Postgres supports binary json up to 255MB. That would not be sufficient if a system card has data up to 2GB. Although 2GB sounds rather large for a system card. (https://www.crunchydata.com/blog/state-of-json-in-postgres-2022)

Reply:
Why do we think it would be in that range? I'd expect system cards to be sub 1MB.